### PR TITLE
style: simplify statements for readability

### LINKED ISSUE #340: Some statements can be simplified for readability
We can replace statements like

```rust
println!("message sent. id: {}, msg: {:?}", sent, msg);
```

with

```rust
println!("message sent. id: {sent}, msg: {msg:?}");
```

to improve code readability and maintainability.

### PR DESCRIPTION
**Pull request by hamirmahal**
Closes #340

### CODE REVIEW

File: pgmq-rs/src/pg_ext.rs [hunk 1]
```diff
@@ -104,8 +104,7 @@ impl PGMQueueExt {
         // we need to check whether the queue exists first
         // pg_partman create operations are currently unable to be idempotent
         let exists_stmt = format!(
-            "SELECT EXISTS(SELECT * from part_config where parent_table = '{queue_table}');",
-            queue_table = queue_table
+            "SELECT EXISTS(SELECT * from part_config where parent_table = '{queue_table}');"
```
Comment (ChuckHend) [inline pgmq-rs/src/pg_ext.rs:107]: since we are in here cleaning this up, can we use a bind parameter instead of string formatting?

**Comment by ChuckHend** [comment]
Thank you! I left a comment in-line regarding a bind parameter.

Also, can you bump the patch ver on [Cargo.toml](https://github.com/tembo-io/pgmq/blob/5eab98308047b477122219bf7df5ad710474f697/pgmq-rs/Cargo.toml#L3)? I think we can release this when it merges.

**Comment by hamirmahal** [comment]
You're welcome! I'll work on both of those changes right now.

File: pgmq-rs/src/pg_ext.rs (hunk 1 above)
Comment (hamirmahal) [reply pgmq-rs/src/pg_ext.rs]: Sure! Let me know if [59bdf9b](https://github.com/tembo-io/pgmq/pull/341/commits/59bdf9ba5cf6bf58da3c5565b48c13c8fab9cdf0) isn't what you had in mind.

Comment (ChuckHend) [reply pgmq-rs/src/pg_ext.rs]: Yes that is exactly what I had in mind. Thank you!

Comment (hamirmahal) [reply pgmq-rs/src/pg_ext.rs]: You're welcome! I'm happy to help.

**Review by ChuckHend — APPROVED** [review]

**Pull request merged**